README.md

```diff
@@ -102,4 +102,46 @@ When you have implemented the two (or more) classes, describe the complexity of
 
 **FIXME: OPERATION COMPLEXITY**
 
+For doubly linked lists: 
+- Accesing is in linear time
+- Searching is in linear time 
+- Adding or deleting elements are in constant time
+
+Function   | ListGenome | LinkedListGenome | LinkedListGenome2
+-----------|------------|------------------|------------------
+init       | O(n)       | O(n)             | O(n)
+insert_te  | O(m+k) + O(disable_te)| O(n+m) + O(disable_te) | O(n+m+k + disable_te)
+copy_te    | O(n) + O(insert_te)| O(n+m) + O(disable_te) | O(n) + O(insert_te)
```

Comment:
There is a $+k$ in the middle, because you search through the TEs. But there should also have been that in the `O(disable_te)` so you could say it goes there.